backend/app/modules/user/user.controller.ts

```diff
@@ -35,23 +35,53 @@ export class UserController {
 
   public async createUser(req: Request, res: Response): Promise<Response> {
     const body: IUser = req.body;
+    const { email, nickname } = body;
+    const isUserExistsByEmail = await this.UserService.getUserByEmail(email);
+    const isUserExistsByNickname = await this.UserService.getUserByNickname(nickname);
```

Comment:
In the future probably we will create some common service method to find user by query
But for now, it's fine